src/test/java/nl/hsac/fitnesse/fixture/slim/JsonHttpTestTest.java

```diff
@@ -6,8 +6,7 @@
 
```

Comment:
Please revert all changes to this file (`*` imports are a bad idea, and I like newlines at the end of my files)